.ruby-version

```diff
@@ -0,0 +1 @@
+2.3.0-dev
```

Comment:
What are your thoughts on removing this and dropping .ruby-version in .gitignore?